package.json

```diff
@@ -54,6 +54,11 @@
       }
     }
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn run lint-fix && yarn run pretty -- --staged && yarn run stylelint"
```

Comment:
@katalonne Change this to yarn run test and edit that for this.

Reply:
@iwarner That's not a good idea we have to run pretty on the staged files. Do you want to change this regardless?

Reply:
Change requested is more like:
"pretty": "prettier-standard -- --staged"
"pre-commit": "yarn run lint-fix && yarn run pretty && yarn run stylelint"

So that our local test is the same as what we run when its lint staged